homeassistant/components/aprilaire/strings.json

```diff
@@ -23,6 +23,35 @@
       "thermostat": {
         "name": "Thermostat"
       }
+    },
+    "sensor": {
+      "indoor_humidity_controlling_sensor": {
+        "name": "Indoor Humidity Controlling Sensor"
```

Comment:
Please use sentence case instead of title case

Reply:
Done